lardataobj/Simulation/ParticleAncestryMap.cxx

```diff
@@ -0,0 +1,41 @@
+#ifndef PARTICLEANCESTRYMAP_CXX
+#define PARTICLEANCESTRYMAP_CXX
+
+#include "lardataobj/Simulation/ParticleAncestryMap.h"
+
+namespace sim {
+
+  void ParticleAncestryMap::SetMap(const std::map<int, std::set<int>>& map) { fParticleMap = map; }
+
+  std::map<int, std::set<int>> ParticleAncestryMap::GetMap() const { return fParticleMap; }
+
+  bool ParticleAncestryMap::HasDroppedDescendants(const int& trackid)
```

Comment:
Here and elsewhere, `int` objects (which are smaller than pointers) should be passed in by value:

```suggestion
  bool ParticleAncestryMap::HasDroppedDescendants(const int trackid)
```